lib/rustler_precompiled.ex

```diff
@@ -319,7 +319,12 @@ defmodule RustlerPrecompiled do
   end
 
   defp tar_gz_urls(base_url, file_name, target_triple, variants) do
-    [lib_name, _] = String.split(file_name, ".", parts: 2)
+    [lib_name, _] =
+      file_name
+      |> String.reverse()
+      |> String.split(".", parts: 2)
+      |> Enum.map(&String.reverse/1)
+      |> Enum.reverse()
```

Comment:
Not the prettiest solution.

Couldn't find a better way to fix this that wouldn't involve a bigger refactor on how tar gz urls are built.
It might be a good option to refactor so variants url could be built from the parts without needed to split some already concatenated string.

Let me know if this workaround is fine for now or you prefer attempt a refactor instead.

Reply:
I cannot take a deeper look right now, but I think we could refactor to keep the parts in a sort of map or struct. WDYT?

Reply:
It might work well.
I'll try something like that...

Reply:
Came up with https://github.com/philss/rustler_precompiled/pull/63, which builds a bit better the urls from parts. Yet no map or struct approach.